config/api.yml

```diff
@@ -861,20 +861,30 @@
     :description: Flavors
     :identifier: flavor
     :options:
-    - :collection
-    :verbs: *gp
+    - :subcollection
```

Comment:
I didn't realize the collection already exists, I am assuming we'll still want to keep the collection actions? And just add in the subcollection actions?

Reply:
👍 Yeah I think if it already exists we can preserve the ability to retrieve flavors using only their id